.github/workflows/test-cluster-delete.yml

```diff
@@ -0,0 +1,39 @@
+# Alert: This workflow will delete "ALL" the ODFE testing cluster stacks
+name: Delete testing clusters
+on:
+  repository_dispatch:
+    types: [test-cluster-delete]
+
+jobs:
+  Create-Cluster:
+    name: Delete Testing cluster
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.STACK_KEY }}
+          aws-secret-access-key: ${{ secrets.STACK_SECRET }}
+          aws-region: us-west-2
+
+      - name: Deleting all ODFE clusters
+        run: |
+            #!/bin/bash
+            set -e
+            stacks=`aws cloudformation list-stacks --stack-status-filter CREATE_COMPLETE --query 'StackSummaries[*].StackName' --output text`
```

Comment:
Should we give a prefix to the test domain stack names in case we mistakenly remove something else?

Reply:
It will only delete the stacks starting with "ODFE-" [here](https://github.com/opendistro-for-elasticsearch/opendistro-build/pull/282/files/ec652d8e2bd2a2ae85cc384b80add3e6745f2645#diff-4196d6509c0ea4f460880b89384f675eR32) What prefix do you suggest?

Reply:
Resolved.